src/components/Root.tsx

```diff
@@ -0,0 +1,8 @@
+import { classnames, margin } from 'classnames/tailwind'
+import ChildrenProp from 'models/ChildrenProp'
+
+const root = classnames(margin('mx-auto'))
+
+export default function ({ children }: ChildrenProp) {
```

Comment:
Actually, it't not a `Root`, it's `InnerRoot` or `InnerWrapper`, something like that

Reply:
Maybe it's better to have such styles inside the `index.css` > `body` 🤔

Reply:
Nah, this won't work